CorePayments/src/main/java/com/paypal/android/corepayments/analytics/AnalyticsEventData.kt

```diff
@@ -19,6 +20,7 @@ data class AnalyticsEventData(
         const val KEY_COMPONENT = "comp"
         const val KEY_DEVICE_MANUFACTURER = "device_manufacturer"
         const val KEY_DEVICE_MODEL = "mobile_device_model"
+        const val KEY_ENVIRONMENT = "merchant_app_environment"
```

Comment:
See this [conversation](https://github.com/paypal/iOS-SDK/pull/119#discussion_r1100695230) for feedback on this tag naming. Will impact both PPCP & BT Arachne.